### PR TITLE
add project files

### PR DESCRIPTION
**Pull request by kazim110**
1. add index.html
2. add style.css
3. add needed files

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  a:visited {
+    color: #74638f;
+  }
+  
+  #main {
+    margin: 30px 8px;
+    padding: 15px;
+    border-radius: 5px;
+    background: #eee;
+  }
```
Comment (kazim110) [inline css/style.css:45]: The main container of the projects has a display:grid, please kindly fix that

File: index.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
```
Comment (kazim110) [inline index.html:9]: It will good to avoid blank lines and space without reason

File: index.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+          being on his
+          <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank">Wikipedia entry</a>.
+        </h3>
+      </section>
+    </main>
+
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <script  src="js/index.js"></script>
```
Comment (kazim110) [inline index.html:114]: You have more than one space between your script tag. considering that will help you to show your professionalism.

**Review by kazim110 — COMMENTED** [review]
# Changes required ♻️ 

Please check my comments below and make required changes.

Submit for a re-review when you're done.

Thanks 😁

**Review by andgarzonmal — APPROVED** [review]
Hi @kazim110 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations  ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**